maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/DebtCollectionService.java

```diff
@@ -0,0 +1,36 @@
+package gov.uk.courtdata.dces.service;
+
+import com.amazonaws.xray.spring.aop.XRayEnabled;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+@XRayEnabled
+public class DebtCollectionService {
+
+    private final DebtCollectionRepository debtCollectionRepository;
+
+    public List<String> getContributionFiles(final LocalDate fromDate, final LocalDate toDate) {
+        log.info("date with the -> {} {}", fromDate, toDate);
+        return debtCollectionRepository.getContributionFiles(convertToCorrectDateFormat(fromDate), convertToCorrectDateFormat(toDate));
+    }
+
+    public List<String> getFdcFiles(final LocalDate fromDate, final LocalDate toDate) {
+        log.info("date with the -> {} {}", fromDate, toDate);
+        return debtCollectionRepository.getFdcFiles(convertToCorrectDateFormat(fromDate), convertToCorrectDateFormat(toDate));
+    }
+
+    String convertToCorrectDateFormat(LocalDate localDate) {
+
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        log.info("Printing before date " + localDate.toString());
+        log.info("Printing after date " + localDate.format(dateTimeFormatter));
```

Comment:
Minor thing, we are already logging out the dates up on line 20, not sure if it's a bit overkill for info logging to log another two lines here for the converted dates. Wonder if it would be best to set these as debug level or just do one info log line after the dates have been converted.

Reply:
Its minor and ignore this for now as we might need to see these in the logs.